src/components/pages/Login/LoginContainer.js

```diff
@@ -8,6 +8,7 @@ const LoginContainer = ({ setCorsErrorModalOpen }) => {
   const { oktaAuth } = useOktaAuth();
 
   useEffect(() => {
```

Comment:
the first item in this destructure was not being used - as I am not sure what it is included for or used for I decided to leave it in there and just asked the linter to ignore it.